### ISSUE TITLE
Watch mode doesn't work on windows

### ISSUE THREAD

**Issue by webmarkelov** [itgalaxy/nunjucks-webpack-plugin#7]
Hi! Watch mode doesn't work on windows.
```
os: windows 10
webpack: 3.8.1
nunjucks-webpack-plugin: 4.0.0
```
[example](https://github.com/webmarkelov/laravel-mix-example)

**Comment by lexcode** [comment]
Hi @webmarkelov do you have an example of how I could read my templates automatic and generate them without specifying them manually in webpack.config?

thanks

**Comment by webmarkelov** [comment]
No, I don’t have. I think it imposible the plugin expects an array with data.

**Comment by evilebottnawi** [comment]
PR welcome!

**Comment by alimzhanovamir** [comment]
> Hi @webmarkelov do you have an example of how I could read my templates automatic and generate them without specifying them manually in webpack.config?
> 
> thanks

Install and run on windows this [webpack config](https://github.com/bnku/webpack-nunjucks-sass-template), to see bug. 
Steps:
1. `npm run dev`;
2. change one of the files .njk.
